data/data/manifests/openshift/baremetal-provisioning-config.yaml.template

```diff
@@ -2,6 +2,8 @@ apiVersion: metal3.io/v1alpha1
 kind: Provisioning
 metadata:
   name: provisioning-configuration
+  annotations:
+    provisioning.metal3.io/paused: "true"
```

Comment:
Maybe leave a comment explaining why we do this?

Reply:
Done